src/uiprotect/data/devices.py

```diff
@@ -1148,6 +1160,23 @@ def last_smart_detect_event(self) -> Event | None:
             return None
         return self._api.bootstrap.events.get(last_smart_detect_event_id)
 
+    @property
+    def last_nfc_card_scanned_event(self) -> Event | None:
+        if (
+            last_nfc_card_scanned_event_id := self.last_nfc_card_scanned_event_id
+        ) is None:
+            return None
```

Comment:
It looks like coverage is missing for this branch

Reply:
@bdraco  should be fixed now

Reply:
@RaHehl Looks like it still showing the `return None` blocks as uncovered.

I merged in `main` as well to give it another run in case codecov was being flakey

Reply:
@bdraco I'm currently checking this again and would tend to agree with you. However, I did everything identically to comparable code and am a bit confused as to how I can achieve the coverage

Reply:
You'll need some test data that doesn't have a fingerprint and nfc event id in it

Reply:
Almost there. 💪  Down to one uncovered line.

Reply:
💪 should be covered now